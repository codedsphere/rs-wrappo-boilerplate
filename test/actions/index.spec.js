import * as reducers from 'mt-action-reducers';
import * as saga from 'mt-action-saga';

describe('actions', () => {
  it('should have the correct actions', () => {
    expect(reducers).toMatchSnapshot();
    expect(saga).toMatchSnapshot();
  });
});
