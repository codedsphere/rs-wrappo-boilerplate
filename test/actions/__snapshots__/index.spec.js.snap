// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions should have the correct actions 1`] = `
Object {
  "default": Object {
    "appModel": Object {
      "initializeAll": [Function],
      "initializeState": [Function],
      "removeState": [Function],
      "updateState": [Function],
    },
  },
}
`;

exports[`actions should have the correct actions 2`] = `
Object {
  "pageSaga": Object {
    "callGitRepo": [Function],
    "getUserLogout": [Function],
    "putUserLogin": [Function],
  },
}
`;
